Sources/FusionBluetooth_Common/BluetoothManager.swift

```diff
@@ -0,0 +1,126 @@
+import Foundation
+
+/**
+ *  @struct Peripheral
+ *
+ *  @discussion The device data for peripheral
+ *
+ */
+public struct Peripheral: Equatable {
+
+    /**
+     *  @property name
+     *
+     *  @discussion The device name.
+     *
+     */
+    public let name: String?
+    
+    /**
+     *  @property uuid
+     *
+     *  @discussion The identifier in iOS and MAC address in Android.
+     *
+     */
+    public let uuid: String
+    
+     /**
+     *  @property isConnected
+     *
+     *  @discussion The state whether the peripheral is connected or not
+     *
+     */
+    public var isConnected: Bool
+    
+    public init(name: String?, uuid: String, isConnected: Bool) {
+        self.name = name
+        self.uuid = uuid
+        self.isConnected = isConnected
+    }
+}
+
+/**
+ *  @protocol BluetoothManagerProtocol
+ *
+ *  @discussion The delegate of managing Bluetooth.
+ *
+ */
+public protocol BluetoothManagerProtocol {
+
+	/*
+     * @method requestAuthorization:
+     *
+     * @discussion Requests the ACCESS_FINE_LOCATION permission. For only Android
+     */
+    func requestAuthorization()
+    
+    /*
+     * @method isScanning:
+     *
+     * @discussion Whether or not the central is currently scanning.
+     */
+	func isScanning() -> Bool
```

Comment:
Is it the same as discovering devices ? If yes, please, rename appropriately

Reply:
yeah I renamed